todo_app/templates/index.html

```diff
@@ -8,27 +8,44 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <form action="/add_item" method="post">
+    <div class="col-md-12"> 
+      <h2>New Item</h2>
+      <form action="{{ url_for('add_todo_item') }}" method="post">
+        <label for="title">Title</label>
         <input class="form-control form-control-lg" type="text" name="title" id="title" placeholder="What do you want to do?" required>
-        <label for="title"><em>Press <code>Return</code> to save</em></label>
+        <label for="description">Description</label>
+        <input class="form-control form-control-lg" type="text" name="description" id="description" placeholder="Perhaps a more detailed description?">
+        <button type="submit" class="btn btn-primary">Add</button>
+        <label for="title"><em>or press <code>Return</code> to save</em></label>
       </form>
-      <ul class="list-group mb-4">
-        {% if not items_list %}
-          <b>No items to do yet</b>
-        {% endif %}
-        {% for item in items_list %}
-          <li>
-            {% if item.status == 'Completed' %}
-              <del><a href="{{ url_for('complete_item', item_id=item.id) }}">{{ item.title }}</a></del>
+
+      <h2>Items</h2>
+      {% if not items_list %}
+        <b>No items to do yet</b>
+      {% endif %}
+      {% for item in items_list %}
+        <div class="row">
+          <div class="col-sm">
+            {% if item.status.value == 'Done' %}
+              <a href="{{ url_for('uncomplete_item', item_id=item.id) }}">
+                <del>{{ item.title }} - {{ item.description }}</a>
```

Comment:
Your closing tag `</a>` doesn't match your opening `<del>` tag - it looks like the browser is adjusting for that, but we should also fix that